include/fc/reflect/typename.hpp

```diff
@@ -92,6 +93,15 @@ namespace fc {
          return n.c_str();
       }
   }; 
+  template<typename T,size_t N> struct get_typename< std::array<T,N> >  
+  { 
+     static const char* name()  
+     { 
+        static std::string _name = std::string("std::array<") + std::string(fc::get_typename<T>::name())
+                                   + "," + fc::to_string(N) + ">";
```

Comment:
Seems like one of the major themes here is replacing `fc::xyz` types with equivalent `std::xyz` types... But here, you still use `fc::to_string` over `std::to_string`. Why keep the fc version?

Reply:
I didn't replace `fc::to_string` with `std::to_string` generically because I wasn't sure if `std::to_string` always produces identical results. I agree that it probably wouldn't matter in this case, but I didn't check individual usages of `fc::to_string`.